src/sources/fluent/mod.rs

```diff
@@ -50,6 +51,11 @@ pub struct FluentConfig {
     #[configurable(derived)]
     keepalive: Option<TcpKeepaliveConfig>,
 
+    /// List of allowed origin IP networks
+    ///
+    /// By default, all origins are allowed
```

Comment:
```suggestion
    #[configurable(derived)]
```

We could take the docs from the `IpAllowlistConfig` to save repetition.